frontend/src/components/App/Home/index.tsx

```diff
@@ -0,0 +1,213 @@
+import { Icon } from '@iconify/react';
+import { useTheme } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import IconButton from '@material-ui/core/IconButton';
+import ListItemText from '@material-ui/core/ListItemText';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import helpers from '../../../helpers';
+import { useClustersConf, useClustersVersion } from '../../../lib/k8s';
+import { ApiError, deleteCluster } from '../../../lib/k8s/apiProxy';
+import { Cluster } from '../../../lib/k8s/cluster';
+import { createRouteURL } from '../../../lib/router';
+import { useFilterFunc, useId } from '../../../lib/util';
+import { setConfig } from '../../../redux/actions/actions';
+import { Link, PageGrid, SectionBox, SectionFilterHeader, SimpleTable } from '../../common';
+import RecentClusters from './RecentClusters';
+
+function ContextMenu({ cluster }: { cluster: Cluster }) {
```

Comment:
This could be in Home/ContextMenu.tsx

There could be stories for this.

Reply:
I'd like that as part of a Home story but it's a bit complex to do it now.